sync-diff-inspector/shard-diff.md

```diff
@@ -6,11 +6,15 @@ aliases: ['/docs/dev/sync-diff-inspector/shard-diff/','/docs/dev/reference/tools
 
 # Data Check in the Sharding Scenario
 
-sync-diff-inspector supports data check in the sharding scenario. Assuming that you have two MySQL instances and now use a replication tool to replicate data into TiDB as shown below:
+sync-diff-inspector supports data check in the sharding scenario. Assume that you use the DM replication tool to replicate data from multiple MySQL instances into TiDB, and now you can use sync-diff-inspector to check upstream and downstream data.
 
-![shard-table-sync](/media/shard-table-sync.png)
+## Use `table-config` for configuration
 
-To configure `table-0` in `table-config`, set `is-sharding=true` and configure the upstream table information in `table-config.source-tables`. Below is a complete example of the configuration.
+You can use `table-config` to configure `table-0`, set `is-sharding=true` and configure the upstream table information in `table-config.source-tables`. This configuration method requires setting all sharded tables, which is suitable for scenarios where the number of upstream sharded tables is small and the naming rules of sharded tables are irregular as shown below.
```

Comment:
I think you'd better cut the sentence "This configuration method requires setting all sharded tables, which is suitable for scenarios where the number of upstream sharded tables is small and the naming rules of sharded tables are irregular as shown below."  into two shorter sentences?

Reply:
I think this sentence is easy to understand. It's okay to leave it.

Reply:
Ok